src/StackAdmin/AzureRM/AzureRM.psd1

```diff
@@ -1,100 +1,103 @@
-﻿#  
```

Comment:
@xizhamsft it looks like these last few files (`AzureRM.psd1`, `AzureRM.psm1` and `AzureStack.psm1`) were updated after a full build -- please revert the changes made to these files

Reply:
@cormacpayne , thanks for the review.  Per your request, reverted those AzureRm/AzureStack file changes and fixed the change log. Please take a look.